packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -229,8 +229,165 @@ export = {
             }
         });
 
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        table.configureReadAutoScaling({
```

Comment:
table.addReadAutoScaling() would be a more consistent name

Reply:
Yep, makes sense. Let's get a second opinion.